### ISSUE TITLE
Ruby Library

### ISSUE THREAD

**Issue by ankane** [Xtra-Computing/thundersvm#181]
Hi, thanks for this awesome library! I wanted to let you know that there are now [Ruby bindings](https://github.com/ankane/thundersvm) for it  🎉  If you have any feedback, let me know or feel free to create an issue on the project for it.

A few notes:

- To make installation easier, I'm hoping to include shared libraries in the gem. There's a link in the readme to [Windows artifacts](https://ci.appveyor.com/project/shijiashuai/thundersvm/branch/master/artifacts), but the latest build is from two years ago and the artifacts are no longer available. I've extracted `thundersvm.dll` from the Windows wheel in the latest release and it appears to work well. If possible, would be great to publish the shared libraries (`dll`, `so`, and eventually `dylib`) with releases.
- Many releases on GitHub (like the most current one - 0.3.4) are marked as "pre-release" in the UI. Is this intentional?

I've also created a Homebrew formula to make installation easier on Mac. Will create a separate issue to discuss that.

**Comment by zeyiwen** [comment]
Thanks Andrew! Another great effort. I have removed the ``pre-release`` tag.

Please feel free to revise the ``README.md`` to add your Ruby interface (e.g., "Support Python, R and Matlab interfaces." -> "Support Python, R, Matlab and Ruby interfaces.") and help fix the old links to Windows artifacts. You may consider contributing your Ruby bindings directly into ThunderSVM--perhaps easier for the future upgrade.

**Comment by ankane** [comment]
Hey @zeyiwen, thanks for the response. Is there a newer link for this line: "pre-built binaries and DLL for Windows x64 on CPUs are avaliable" or should it be removed?

Re Ruby bindings: I think it'll be easier to manage as a separate project.

**Comment by ankane** [comment]
Also, fwiw, it looks like there are installation instructions on 3 separate pages.

- README.md
- docs/get-started.md
- docs/how-to.md

It'd be nice if a maintainer could help address these as well.

**Comment by shijiashuai** [comment]
CI on windows has been migrated to Travis-ci, so the “windows artifacts”
link can be removed.

Releasing shared library is a good idea. While building on MacOS is on
progress, I’ll update the release once it is finished.
